src/components/anchor-navigation/anchor-navigation.pw.tsx

```diff
@@ -0,0 +1,145 @@
+      mount,
+      page,
+    }) => {
+      await mount(<AnchorNavigationComponent />);
+
+      await anchorNavigationStickyNavigation(page, sectionIndex).click();
+      await expect(
+        anchorNavigationStickyMainPage(page, sectionName)
+      ).toBeVisible();
+    });
+  });
+
+  [
+    ["First", "First section"],
+    ["Fifth", "Fifth section"],
+  ].forEach(([sectionIndex, sectionName]) => {
+    test(`should scroll to the ${sectionIndex} and verify that proper ${sectionName} AnchorNavigation row is visible`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<AnchorNavigationComponent />);
+
+      await anchorNavigationStickyNavigation(page, sectionIndex).click();
+      await anchorNavigationStickyMainPage(page, sectionName).focus();
+      await expect(
+        anchorNavigationStickyMainPage(page, sectionName)
+      ).toBeVisible();
+    });
+  });
+});
+
+test.describe("When focused", () => {
+  test("has the expected styling when the focusRedesignOptOut flag is true", async ({
+    mount,
+    page,
+  }) => {
+    await mount<HooksConfig>(<AnchorNavigationComponent />, {
+      hooksConfig: { focusRedesignOptOut: true },
+    });
+
+    await anchorNavigationItem(page, 0).focus();
+
+    await expect(anchorNavigationItem(page, 0)).toHaveCSS(
+      "outline",
+      "rgb(255, 188, 25) solid 3px"
+    );
+  });
+
+  test("has the expected styling when the focusRedesignOptOut flag is false", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<AnchorNavigationComponent />);
+
+    const anchorNavItem = anchorNavigationItem(page, 0);
+
+    await anchorNavItem.focus();
+    await expect(anchorNavItem).toHaveCSS(
+      "box-shadow",
+      "rgb(255, 188, 25) 0px 0px 0px 3px, rgba(0, 0, 0, 0.9) 0px 0px 0px 6px"
+    );
+
+    await expect(anchorNavItem).toHaveCSS(
+      "outline",
+      "rgba(0, 0, 0, 0) solid 3px"
+    );
+  });
+});
+
+test.describe("Rounded corners", () => {
+  test("should have the expected border radius styling", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<AnchorNavigationComponent />);
+
+    await expect(anchorNavigationItem(page, 0)).toHaveCSS(
+      "border-radius",
+      "0px 8px 8px 0px"
+    );
+    await expect(anchorNavigationItem(page, 1)).toHaveCSS(
+      "border-radius",
+      "0px 8px 8px 0px"
+    );
+    await expect(anchorNavigationItem(page, 2)).toHaveCSS(
+      "border-radius",
+      "0px 8px 8px 0px"
+    );
+    await expect(anchorNavigationItem(page, 3)).toHaveCSS(
+      "border-radius",
+      "0px 8px 8px 0px"
+    );
+    await expect(anchorNavigationItem(page, 4)).toHaveCSS(
+      "border-radius",
+      "0px 8px 8px 0px"
+    );
+  });
+});
+
+test.describe("Accessibility tests for Anchor Navigation component", () => {
+  test("should pass accessibility tests for Anchor Navigation default story", async ({
```

Comment:
```suggestion
  test("should pass when in default composition", async ({
```

or something similar 🤷